pcdsdevices/lxe.py

```diff
@@ -210,3 +212,27 @@ def move(self, position, wait=True, timeout=None, moved_cb=None):
     def wm(self) -> float:
         # Remove the PseudoPosition tuple for FltMvInterface compatibility
         return super().wm[0]
+
+
+class LaserEnergyTimingLxt(PVPositioner, FltMvInterface):
```

Comment:
I think this deserves at least a quick look from @silkenelson to confirm that this class name makes sense

Reply:
Name and high-level functionality if possible, too! (@silkenelson)